WalletWasabi/Wallets/Wallet.cs

```diff
@@ -319,10 +326,10 @@ public async override Task StopAsync(CancellationToken cancel)
 					ChaumianClient.OnDequeue -= ChaumianClient_OnDequeue;
 
 					await ChaumianClient.StopAsync(cancel).ConfigureAwait(false);
-					Logger.LogInfo($"{nameof(ChaumianClient)} is stopped.");
 				}
 
 				DisconnectDisposeNullLocalBitcoinCoreNode();
+				Logger.LogInfo($"{nameof(ChaumianClient)} is stopped.");
```

Comment:
??? This is not good.

Reply:
Oops this is my fault too. I didn't see the log is outside of the ''if' block.